spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -236,6 +236,15 @@ def ensure_version!(version_number, platform: nil)
         end
       end
 
+      def reject_version_if_possible!
+        can_reject = edit_version.can_reject_version
+        if can_reject
```

Comment:
I believe we will still get error saying `Problem processing review submission.` if the app is not in 'rejectable' state.

Reply:
Is that ok?

Reply:
I think this is okay the way it is now for that situation ^